content/news/buzz/march-2024-status-update/contents.lr

```diff
@@ -0,0 +1,69 @@
+title: March 2024 Status Update
+---
+author: Russell Keith-Magee
+---
+body:
+
+This month, we have less to report by raw feature count - but the changes we *have* made represent extremely significant progress.
+
+What we've done
+----------------
+
+* Our primary focus this month has been making the changes to CPython needed to add support for `iOS <https://github.com/python/cpython/issues/114099>`__ and `Android <https://github.com/python/cpython/issues/116622>`__. We've made major progress towards this goal: all the patches required for iOS have been either merged, or have undergone significant review from the CPython core team; a large number of patches have been submitted for Android, with only a small number still required. This month, we have:
```

Comment:
Late breaking news - it looks very likely that the last iOS patches will land before I need to post this; if so:

```suggestion
* Our primary focus this month has been making the changes to CPython needed to add support for `iOS <https://github.com/python/cpython/issues/114099>`__ and `Android <https://github.com/python/cpython/issues/116622>`__. We've made major progress towards this goal: all the patches required for iOS have been merged; a large number of patches have been submitted for Android, with only a small number still required. This month, we have:
```